packages/utilities/gel-styled-components/spacings.js

```diff
@@ -0,0 +1,20 @@
+/*
+    GEL Spacing
+    Assumes 16px is set as the default font-size.
+    This is changeable in the user's browser settings, as the html font-size is 100%
```

Comment:
As part of this package, we should include in the `README.md` file the decisions we made with regards to using `rem` instead of `em` and how we recommend using these utilities within an application. 

https://github.com/BBC-News/simorgh/blob/latest/docs/Spacing-Units.md

We should also explain our usage in `/typography.js` of `em` for `font-size` and `rem` for `line-height`. 

Since this is blocking other tickets, I'd be happy for this to be added in another Issue/PR.